### DISCUSSION TITLE
iPhone 戴口罩解鎖 Face ID

### DISCUSSION THREAD

**Discussion by HuanYuuuuuu** [1101-GS4719-C/Python-Community-GS4719-C#481] [General]
不知道大家會不會跟我一樣之前因為疫情要一直戴著口罩，所以常常不能用Face ID。又不好意思在人多的時候拿下口罩來解鎖。 而近期蘋果公司推出了新的口罩解鎖，正好可以解決這一問題。但同時也讓我思考戴著口罩解所會不會讓跟我很像的人也可以解鎖我的手機?
根據蘋果提供的文件指出，「口罩 Face ID」主要是偵測眼睛周圍的臉部特徵來進行辨識、解鎖 iPhone，而且即便戴著眼鏡都可以成功辨識。雖然依照蘋果的對客戶隱私 權力極為嚴謹的作風來看，感覺能推出到市面上就應該不會有上述的問題。
但我自己其實很好奇雙胞胎能不能解鎖對方的手機，不知道大家有沒有聽過類似的案例
不過我自己有遇到如果帶著帽子或是安全帽的話，口罩Face ID就不能順利解鎖的問題。
(剛好我平常都會帶帽子出門，結果想要戴著口罩解鎖還是得拿下帽子.....)

![image](https://user-images.githubusercontent.com/99785986/158581848-e5be9718-9ccc-42fe-bc69-c14cc03608e1.png)

**Comment by ThomasMeng0515** [comment]
雖然蘋果是說安全性沒有疑慮，但是感覺沒有用全臉解鎖，安全性就低一些(?
目前還沒有更新到iOS15.4，之後可以來試試看~

**Reply by HuanYuuuuuu** [reply]
是時候玩玩看口罩解鎖了 真的蠻方便的

**Comment by sean1325418** [comment]
感覺戴口罩後的臉部特徵會少很多，這樣精準度不免會讓人懷疑

**Comment by dayu1004** [comment]
之前系上在上光學系統相關課程的時候，有提到臉部辨識是利用光學的傅氏平面去計算每個人的面部特徵。所以我想即便是戴了口罩，眼部及額頭的輪廓應該每個人都會不相同。即使肉眼上雙胞胎兩個人長得很相似，實際上他們的面部結構跟面高可能都會足以讓光學系統辨識的差距~~

**Reply by yi-1111** [reply]
哦~原來是利用光學系統，又學到一課了

**Reply by Rosancl** [reply]
長知識了！原來是應用到光學去辨識~~

**Comment by 11timyyy** [comment]
能辨識的地方少了很多，希望能精準地辨識出主人。

**Comment by yuhao112** [comment]
其實口罩辨識還是沒有到那麼方便的，首他只能認一張臉孔，再來他好像要平視什麼的他才能順利地解鎖鎖

**Comment by yucheng1028** [comment]
人臉辨識功能真的變越來越強大了，雖然精確度還不曉得

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望人臉辨識可以普及化。

**Comment by bloodtrail0511** [comment]
這個功能終於要來了，算是不知道何時結束的疫情中的小確幸吧

**Comment by xkllkx** [comment]
謝謝分享，但我覺得還是有安全性的疑慮。

**Comment by chupochun** [comment]
我覺得這個功能對我來說真的很方便，我對apple的安全性也很有信心

**Comment by cordydeng** [comment]
口罩解鎖感覺在疫情時代真的會讓臉部辨識更加方便且安全，可惜的是同樣是用臉部辨識的ipad pro 卻還沒有這樣的功能，是還蠻可惜的

**Comment by Sazmafuji** [comment]
大家，如果你不是iphone12之後的，你們示沒人權的，因為沒支援QQ，我就是

**Reply by awkward-willy** [reply]
真的...，我也是其中之一
不過也已經習慣按密碼了，速度上其實沒有差很多，只是體驗真的沒那麼好

**Comment by hapiwangy** [comment]
對於防疫時期的現在，這真的是人們的一大福音。不果我覺得指紋辨識相比較起來也比較靠普一點

**Comment by lx3211** [comment]
要是iphone也可以搭載touch id也不用擔心口罩face id造成的安全疑慮了

**Comment by PoooYen** [comment]
蘋果.....終於....做事了......，擠牙膏擠到現在終於擠出一點點了

**Comment by henrywen547** [comment]
這裡有個還在用Touch ID的人...
窩好期待i14啊啊啊

**Comment by ghost** [comment]
目前對這個技術的進步空間個人是保持樂觀的，不過確實是否會對識別門檻過低這點是值得思考的

**Comment by HaoFish123** [comment]
感謝分享，戴口罩要解鎖IPHONE變得更容易了

**Comment by Yuhsin000** [comment]
不知道ipad有沒有這個更新，來去看看!

**Comment by etin2468** [comment]
ipad 好像還沒有這個更新，希望能早點更新，但是耶希望疫情趕快結束，這樣就不需要一直戴口罩了

**Comment by treeann** [comment]
好讚的分享 我馬上來試試 希望資料轉移不會有問題~

**Comment by HuanYuuuuuu** [comment]
最近瀏海變長到會蓋住眉毛的區域，我發現也會影響到口罩Face ID的辨識

**Comment by Lai27330** [comment]
目前使用口罩解鎖的心得是精準度仍有待提升，有時候順手拿起就解鎖了有時認真對著臉卻反而解不開，而且沒有設定戴眼鏡辨識有時也可以解鎖，所以有點困惑，但有這樣跟進時事的設計還是覺得很棒，希望辨識功能再更明確一點就好了。

**Comment by minniewei** [comment]
可惡啊！！！對於非apple使用者來說真是太羨慕了

**Comment by Yao900108** [comment]
用到現在，其實準確性還有待加強。期待之後的進步!!

**Comment by Jay-722** [comment]
從習慣直接臉部解鎖，後來習慣打密碼，現在又要重新習慣直接臉部

**Comment by HaoFish123** [comment]
謝謝分享，目前還未使用過，但感覺蠻有趣的。

**Comment by jason79461385** [comment]
iphoneX沒有支援....本來很期待可以用的說 要不然人在外面的時候 就都要自己輸入密碼

**Comment by treeann** [comment]
我的ipad沒支援...但謝謝樓主分享

**Comment by Jimmy911218** [comment]
謝謝分享但感覺準確率會不會下降呢

**Comment by evanchou1204** [comment]
有時候口罩解鎖偵測不到，還會卡一下才讓我用密碼解鎖，反而花更多時間QQ

**Comment by Yunhans** [comment]
口罩解鎖真的方便，但是角度要拿對不然判斷會不穩定

**Comment by wenchi9023** [comment]
因為實習的關係手機裏頭有公司資料，不得不設密碼，幸好有臉部辨識能更快速解鎖! 加上口罩解鎖功能感覺就超方便啊

**Comment by Pietra1226** [comment]
太難過了，我的手機更新版本沒辦法支援，不然對現在還要戴口罩的我好不方便阿...

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
個人認為 雖然face I'd是最新的科技 但I touch 才是我認爲最方便的解鎖方式

**Comment by Evan0979** [comment]
光憑眼睛就能辨識一個人嗎,好猛

**Comment by CPCpyt** [comment]
我覺得人眼本身就是很難被複製的特徵，
不過我想用更多參數(鼻子嘴巴)來保護自己的密碼應該比較保險。
